src/server/pfs/pretty/pretty.go

```diff
@@ -68,19 +67,19 @@ func NewPrintableRepoInfo(ri *pfs.RepoInfo) *PrintableRepoInfo {
 }
 
 // PrintDetailedRepoInfo pretty-prints detailed repo info.
-func PrintDetailedRepoInfo(repoInfo *PrintableRepoInfo) error {
+func PrintDetailedRepoInfo(w io.Writer, repoInfo *PrintableRepoInfo) error {
 	template, err := template.New("RepoInfo").Funcs(funcMap).Parse(
 		`Name: {{.Repo.Name}}{{if .Description}}
 Description: {{.Description}}{{end}}{{if .FullTimestamps}}
 Created: {{.Created}}{{else}}
 Created: {{prettyAgo .Created}}{{end}}
 Size of HEAD on master: {{prettySize .Details.SizeBytes}}{{if .AuthInfo}}
-Access level: {{ .AuthInfo.AccessLevel.String }}{{end}}
+Access level: {{range .AuthInfo.Permissions}}{{.String}} {{end}}{{end}}
```

Comment:
Yep, this was crashing before because the template was accessing a field that doesn't exist.  If only we had static type-checking as a first-class language feature.